README.md

```diff
@@ -306,52 +306,54 @@ try {
 }
 ```
 
-To enable caching for the requests, one can add `expiresIn` to the requests configuration object. The values are in seconds.
-```
-// cache is valid for 30 minutes
-const requestsConfig = {
-  cache: {
-    expiresIn: 1800,
-  }
-};
-```
-
-To disable caching for the requests that are enabled by default, you can assign 0 to `expiresIn`.
-```
+To enable caching for the requests, one can add `expiresIn` to the requests configuration object. The values are in seconds. Value `0` disables caching.
+```javascript
+// cache is valid for 30 minutes:
 const requestsConfig = {
   cache: {
     expiresIn: 1800,
   }
 };
 ```
 
-Requests can be cached to [CACHE_API](https://developer.mozilla.org/en-US/docs/Web/API/Cache) or to memory, where the target can be specified with.
-```
+Requests can be cached to [Cache API](https://developer.mozilla.org/en-US/docs/Web/API/Cache) or to memory:
+```javascript
   const requestsConfig = {
     cache: {
       expiresIn: 5000,
       targets: [CacheTarget.CACHE_API],
     },
   };
 ```
-```
+
+```javascript
```

Comment:
Do these two examples look better when separate or can they be in the same code block?

Reply:
I agree it did look a bit weird, so I added another sentence in between. Thanks, resolving.

Reply:
Thanks, looks good to me